holoviews/core/data/pandas.py

```diff
@@ -16,7 +18,27 @@
 @lru_cache(maxsize=None)
```

Comment:
Wondering if this is actually needed. Aren't warnings generally just shown once? [Python docs](https://docs.python.org/3.10/library/warnings.html) says that "Repetitions of a particular warning for the same source location are typically suppressed.".

Reply:
`lru_cache` can be removed. 

The main reason for it being there (in my mind at least) is not to "spam" `hvplot` users (example without no `lru_cache`):
![image](https://user-images.githubusercontent.com/19758978/195023898-213964ce-9d09-4cc7-bbee-42ea7f419d17.png)

All these spams arise because it is a new location for each call. If the location is the same, the warning will only be raised once instead of ten times.
 
![image](https://user-images.githubusercontent.com/19758978/195026522-64e89819-4c87-4eb2-a115-4ca655398735.png)

Reply:
Let's keep it then, that looks horrible otherwise!